app/db_type.py

```diff
@@ -0,0 +1,2 @@
+from sqlalchemy import *  # noqa: F401,F403
```

Comment:
Let's please document the purpose of this file with a docstring, explaining why this is necessary.